src/swapd/runtime.rs

```diff
@@ -81,8 +84,12 @@ pub fn run(
     };
 
     let init_state = match local_swap_role {
-        SwapRole::Alice => State::Alice(AliceState::StartA { local_trade_role }),
```

Comment:
`TradeRole` is `Copy` so we could probably simplify with `*local_trade_role` here.